docs/UserGuide.adoc

```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sorting all persons : `sort`
+
+Sorts the list of all persons, then list them, along with their non-private details, in the address book. +
```

Comment:
Nice work on following the format of the user guide! 

It might also be good to indicate *how* are the persons sorted e.g. Name, Email etc